libs/electron/src/services/electronPlatformUtils.service.ts

```diff
@@ -87,6 +87,12 @@ export class ElectronPlatformUtilsService implements PlatformUtilsService {
     return ipcRenderer.invoke("appVersion");
   }
 
+  getApplicationVersionNumber(): Promise<string> {
+    return ipcRenderer
+      .invoke("appVersion")
+      .then((version) => version.split(RegExp("[+|-]"))[0].trim());
+  }
```

Comment:
I think this should work?

```suggestion
  async getApplicationVersionNumber(): Promise<string> {
    return await (this.getApplicationVersion()).split(/[+|-]/)[0].trim());
  }
```